code/modules/atmospherics/machinery/components/electrolyzer/electrolyzer.dm

```diff
@@ -147,10 +148,6 @@
 
 /obj/machinery/electrolyzer/default_unfasten_wrench(mob/user, obj/item/wrench, time)
 	. = ..()
-	if(anchored)
-		update_use_power(ACTIVE_POWER_USE)
-	else
-		update_use_power(NO_POWER_USE)
```

Comment:
why're you deleting this?

Reply:
Also, don't leave an empty proc with just ``. = ..()``, just delete it.

Reply:
use_power is used for static power usage, which means that if use_power is set to ACTIVE_POWER_USE, it always uses BASE_MACHINE_ACTIVE_CONSUMPTION even when its not on